### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by derekchiang**
The README used to explain what Pachyderm is and what's exciting about it.  I think that we want people who find us on Github to be able to learn what it is without going to our website.  After all, Pachyderm is an OSS project, and a lot of OSS contributors shy away from projects that seem too commercial, if that makes sense.  I'm not even too sure about putting the hiring section on README, but we can talk about that later.

### CODE REVIEW

**Comment by derekchiang** [comment]
@JoeyZwicker could you review?

File: README.md [hunk 1]
```diff
@@ -2,25 +2,75 @@
...
+history of your data, it's much easier to collaborate with teammates, and if
+anything goes wrong you can revert _the entire cluster_ with one click!
+
+Version control is also very synergistic with our containerized processing
+engine. Pachyderm understands how your data changes and thus, as new data
+is ingested, can run your workload on the _diff_ of the data rather than the
+whole thing. This means that there's no difference between a batched job and
+a streaming job, the same code will work for both!
+
+### The Vision
```
Comment (JoeyZwicker) [inline README.md]: remove vision section for now

File: README.md [hunk 2]
```diff
@@ -2,25 +2,75 @@
 [![GitHub release](https://img.shields.io/github/release/pachyderm/pachyderm.svg?style=flat-square)](https://github.com/pachyderm/pachyderm/releases)
 [![GitHub license](https://img.shields.io/github/license/pachyderm/pachyderm.svg?style=flat-square)](https://github.com/pachyderm/pachyderm/blob/master/LICENSE)
 
-* [News](#news)
 * [Getting Started](http://pachyderm.readthedocs.io/)
+* [What is Pachyderm?](#what-is-pachyderm)
+* [What's new about Pachyderm? (How is it different from Hadoop?)](#whats-new-about-pachyderm-how-is-it-different-from-hadoop)
+* [The Vision](#our-vision)
```
Comment (JoeyZwicker) [inline README.md]: remove

**Comment by JoeyZwicker** [comment]
LGTM

**Pull request merged**